src/components/ui/passwordValidationItem.tsx

```diff
@@ -0,0 +1,37 @@
+import { cn } from '@/lib/utils';
+import { CircleCheck } from 'lucide-react';
+
+interface PasswordValidationItemProps {
+  text: string;
+  isValid: boolean;
+  className?: string;
+}
+
+/**
+ * @example
+ * ```ts
+ * return (
+ * <PasswordValidationItem
+ * isValid={false}
+ * text={'Less than or equal to 8 characters long'}
+ * />
+ * )
+ * ```
+ */
+
+export function PasswordValidationItem({
+  text,
+  isValid = false,
+}: PasswordValidationItemProps) {
+  return (
+    <p className={cn('flex', { 'items-center text-sm gap-1': !isValid })}>
```

Comment:
If you want the className to be applied regardless of the `isValid` value, you need to write it in the first argument. `item-center` and `gap-1` are not applied if `isValid` is true.
<img width="549" alt="image" src="https://github.com/Tascurator/tascurator-frontend/assets/144519746/2a5611e0-b0ba-491e-8232-6bffe1c10f4b">